src/components/file-explorer/components/file-explorer-collapsible-sidbar/file-explorer-collapsible-sidebar.component.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from "react";
+
+import IconButtonComponent from "@/components/icon-button/icon-button.component";
+import TextFieldComponent from "@/components/text-field/text-field.component";
+import TypographyComponent from "@/components/typography/typography.component";
+
+import styles from "./file-explorer-collapsible-sidebar.module.css";
+
+type Props = {
+  isOpen: boolean;
```

Comment:
معمولاً واسه booleanهایی که تو Props میگیریم از is استفاده نمیکنیم
و به صورت Optional تعریفشون میکنیم
```suggestion
  open?: boolean;
```